apps/nextjs-website/src/app/[productSlug]/guides/[...productGuidePage]/page.tsx

```diff
@@ -81,7 +77,7 @@ const Page = async ({ params }: { params: Params }) => {
     params?.productSlug,
     params?.productGuidePage ?? ['']
   );
-
+  const urlReplaceMap = (await getUrlReplaceMapProps()) || {};
```

Comment:
I think that ` || {}` is not necessary if you return the staticUrlReplaceMap in getUrlReplaceMapProps function
```suggestion
  const urlReplaceMap = await getUrlReplaceMapProps();
```